staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -239,6 +266,20 @@ func clearSecurityContext(p *corev1.Pod, containerName string) {
 	})
 }
 
+// setPrivileged configures the containers as privileged.
+func setPrivileged(p *corev1.Pod, containerName string) {
+	podutils.VisitContainers(&p.Spec, podutils.AllContainers, func(c *corev1.Container, _ podutils.ContainerType) bool {
+		if c.Name != containerName {
+			return true
+		}
+		if c.SecurityContext == nil {
+			c.SecurityContext = &corev1.SecurityContext{}
+		}
+		c.SecurityContext.Privileged = pointer.BoolPtr(true)
```

Comment:
nit: `BoolPtr` is deprecated, `pointer.Bool` is recommended.

Reply:
followup PR: https://github.com/kubernetes/kubernetes/pull/115837